kitsune/search/v2/base.py

```diff
@@ -1,25 +1,27 @@
 from datetime import datetime
+
 from django.utils import timezone
 from elasticsearch_dsl import Document as DSLDocument
-from elasticsearch_dsl import InnerDoc, field, MetaField
+from elasticsearch_dsl import InnerDoc, MetaField, field
 
 
 class SumoDocument(DSLDocument):
     """Base class with common methods for all the different documents."""
 
-    indexed_on = field.Date()
+    updated_on = field.Date()
```

Comment:
IMO this feels too similarly named to the `updated` field we have on most documents, coming from db. I liked `indexed_on` because it put it clearly in the ES domain. I think, even when we're calling the update API, we still are indexing data. Perhaps `indexed_or_updated_on`?